src/routes/home/HomeView.vue

```diff
@@ -211,8 +208,13 @@ export default {
   async created() {
     await this.gameListStore.requestGameList();
     this.loadingData = false;
+    this.tab = this.$route.path;
   },
   methods: {
+    gameListWindow(url){
+      this.$router.replace(url);
```

Comment:
An awkward thing about using `.replace()` instead of `.push()` is that using the back button won't bring you back to the previous page. If we use `$router.push()` is the rest of the UX significantly different?